python_modules/dagster/dagster/_utils/schedules.py

```diff
@@ -110,7 +112,7 @@ def _replace_date_fields(
             minute,
             0,
             0,
-            tz=pendulum_date.timezone_name,
+            tz=pendulum_date.tzname(),
             dst_rule=TRANSITION_ERROR,
```

Comment:
i think one place you are going to run into trouble is here - "dst_rule" is a pendulum-specific thing that we use to navigate dst boundaries correctly. It's possible we could pull out that logic ourselves though and fork it